libevm/VMValidate.cpp

```diff
@@ -36,22 +40,23 @@ void VM::validate(ExtVMFace& _ext)
 	m_ext = &_ext;
 	initEntry();
 	size_t PC;
-	byte OP;
-	for (PC = 0; (OP = m_code[PC]); ++PC)
-		if  (OP == byte(Instruction::BEGINSUB))
+	Instruction OP;
```

Comment:
Changes for SIMD (and some previous ones) broke VMValidate, so I wanted it to compile again.  Changes are pretty much small stuff like changed signatures and casts.